lib/barnes/instruments/puma_instrument.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'barnes/instruments/puma_stats_value'
+module Barnes
+  module Instruments
+    class PumaInstrument
```

Comment:
Is `Instrument` even necessary given the module name?

Reply:
I was running into namespace issues with `Puma` where I needed to use `::Puma` so it was easier for me to make it not the same constant name as another gem.

Reply:
:+1: